wezterm-ssh/Cargo.toml

```diff
@@ -20,15 +23,21 @@ log = "0.4"
 portable-pty = { version="0.5", path = "../pty" }
 regex = "1"
 smol = "1.2"
-ssh2 = {version="0.9", features=["openssl-on-win32"]}
-#ssh2 = {version="0.9"}
+# ssh2 = {version="0.9", features=["openssl-on-win32"]}
+ssh2 = {version="0.9"}
 
 # Not used directly, but is used to centralize the openssl vendor feature selection
 async_ossl = { path = "../async_ossl" }
 
 [dev-dependencies]
+assert_fs = "1.0.4"
+indoc = "1.0.3"
 k9 = "0.11.0"
+once_cell = "1.8.0"
+predicates = "2.0.2"
```

Comment:
I tend to use just `"2.0"` when specifying deps, as the `"2.0.2"` form doesn't actually pin to `2.0`, and every couple of weeks I run `cargo update` when reminded by dependabot, so the chances of these crates actually being `2.0.2` in practice is very low.

Reply:
@wez I do this out of habit given that even though a library promises that nothing breaks on a patch revision, you may assume some functionality or behavior on the first version you use (e.g. 2.0.2), which if someone else pulls in the minimum version (e.g. 2.0.0) the crate may fail to compile or worse behave unexpectedly. This always guarantees 2.0.2 or above for patch revisions. Is there a reason you want 2.0 specifically?

Did you still want me to switch?

Reply:
Unless you know that the thing you need is in `2.0.2` and not in any lower version, it's just a bit less cluttered to use `2.0`.
I tend only to be more specific in the case that something is broken in a given version of a dep.

Reply:
Updated deps to just use major + minor.